packages/smooth_app/lib/resources/app_food_icons.dart

```diff
@@ -0,0 +1,211 @@
+part of 'app_icons.dart';
+
+class FoodIcons extends AppIcon {
+  const FoodIcons.additives({
+    super.color,
+    super.size,
+    super.shadow,
+    super.semanticLabel,
+    super.key,
+  }) : super._(_FoodIconsFont.additives);
```

Comment:
All that is very very verbose, for no added value.
What about one single constructor instead?
```suggestion
  const FoodIcons({
    super.color,
    super.size,
    super.shadow,
    super.semanticLabel,
    super.key,
    required FoodIconFont icon,
  }) : super._(icon);
```

Reply:
I admit there's a lot of code, but it allows us to have `const` widgets directly.